help/en/html/hardware/loconet/Addressing.shtml

```diff
@@ -131,25 +134,29 @@
           "Turnouts", "Sensors", etc.) by selecting the "File" menu item and then
           selecting "Store", then either "Store configuration only to File..." or
           "Store Configuration and Panels to File...".</p>
+
       <p>JMRI provides a few different ways to automatically load a "panel file"
           at JMRI start-up.  The most-commonly used mechanism is via JMRI's "Start Up"
           "Preferences", which can be accessed via the JMRI "File" menu, then selecting
           "Start Up".  Simply select "Add", then "Open file...", and then select the
           file (or, if required, multiple files) to be opened at start-up.  Then
           select "Open".  It is then necessary to "save" the configuration profile
           to configure JMRI to use the new settings on next JMRI start-up.</p>
+
       <p>On subsequent JMRI invocations, all start-up actions are invoked,
           including any "start-up" actions you have defined to "Load" a "Panel File".</p>
+
       <p>If automatic reloading of the table entries is not desired, it is possible
           to manually read a panel file.  Under the JMRI "Panels" menu item, select
           "Open Panel..." and then locate and select one or more panel files
           containing the appropriate information.  Select the "Open Panels" button
           to read those files into JMRI.</p>
+
       <p>Note that JMRI panel files are typically specific to a given system
           connection.  If you have configured JMRI for multiple system connections,
           the panel file will typically be available only for the system connection
           in which it was created.</p>
-
+-->
```

Comment:
Ok to remove all the above if you think people will actually click through to the new LoadStoreWork page.

Reply:
Done.  Removed the text, some of which had outdated or wrong content.